test/ComplexTest.jl

```diff
@@ -0,0 +1,53 @@
+using ForwardDiff: Dual
```

Comment:
ForwardDiff wraps all test files into a module.

Reply:
Yes, interesting design. May I know the reason to put tests in a module?

Reply:
I have updated the tests. Good now?

Reply:
It won't introduce variables to `Main`, in this way. So tests don't influence each other. This is the motivation for https://github.com/YingboMa/SafeTestsets.jl, too.